scripts/pinned-tools.sh

```diff
@@ -83,3 +83,7 @@ function isDocsBranch() {
   fi
   return 1
 }
+
+function getAcceleratorArgs() {
+  echo ${ACCELERATOR_ENDPOINT:+-adb2 -enableCacheServer -cacheServerEnableDownload true -cacheServerEnableUpload true -cacheServerEndpoint "${ACCELERATOR_ENDPOINT}"}
```

Comment:
I spent a day and a half figuring out why nothing was working until I stumbled upon a forum post where someone says to add `-adb2` if trying to access Accelerator in batch mode

:tableflip:

Reply:
Could it be some of our projects have not been converted to asset db 2?

Reply:
I observed this on all projects

Reply:
We did do the asset db upgrade & then a full asset refresh during the 2019.3 upgrade though.

Reply:
Fresh clones also show asset db 2 is used in the preferences